lib/Alien/Build/Plugin/Build/Autoconf.pm

```diff
@@ -234,6 +238,67 @@ Some reasonable default flags will be provided.
   $self;
 }
 
+sub _pkgconf_wrapper {
+  my ($self, $build) = @_;
+
+  return if !_win;
+
+  my $pk = File::Which::which ($ENV{PKG_CONFIG})
+        || File::Which::which ('ppkg-config')
+        || File::Which::which ('pkg-config.pl')
+        || File::Which::which ('pkg-config');
+
+  if (!defined $pk) {
+    $build->log ("Could not locate ppkg-config or pkg-config in your path:\n");
+    return;
+  }
+  
+  
+  if ($pk =~ /\.bat$/i) {
+    $pk =~ s/\.bat$//i;
+    #  should not die?
+    die "bat file does not have adjacent pkg-config"
+      if !-e $pk;
+  }
+  
+  #  flaky means of determining we are using a perl pkg-config
+  my $data = Path::Tiny->new($pk)->slurp;
+  my $str = <<"EOSTRING"
+# lightweight no-dependency version of pkg-config. This will work on any machine
+# with Perl installed.
+EOSTRING
+  ;
+
+  my $fname;  
+  if ($data =~ /$str/msi) {
+    my $perl = $^X;
```

Comment:
`$perl` could have spaces in it.  It would be nice to support that if it isn't too hard.

Reply:
In fact if `$perl` does then likely `$pk` does too.

Reply:
Good point.  Will surround them in double quotes or escape any spaces.

Reply:
Done in 1fced2874b7bc467857980cc5e20f2355d276190